DOC: exmouth gulf (#74)
jbathmann/pyMANGA: web/content/en/docs/Sample_Model:_Exmouth_Gulf/Installation_exmouth.md

> ---

Reviewer: s.o.

---

> title: "Do it yourself ARBEITSTITEL"
> linkTitle: "Do it yourself ARBEITSTITEL"
> weight: 4

Reviewer: Please update

```suggestion
title: "Do it yourself"
linkTitle: "Do it yourself"
weight: 4
```

description:
---

The model setup of the "Full Model" is located in the following folder in the current version of pyMANGA:

	./Benchmarks/Exmouth_Gulf/full_model

> In order to start a model run with the setup on your own computer, file paths references to input files have to be adjusted in the XML control file. All file paths in the control file are specified absolutely, so each path starts with "/home/..." under Ubuntu and with the letter of the drive under Windows. If the location of the setup is not changed, the control file is located under the following path:

Reviewer: ```suggestion
In order to start a model run with the setup on your own computer, file path references to input files have to be adjusted in the XML control file. All file paths in the control file are absolute, so each path starts with "/home/..." under Ubuntu and with the letter of the drive under Windows. If the location of the setup is not changed, the control file is located under the following path:
```

Author: Done

	./Benchmarks/Exmouth_Gulf/full_model/setup_pymanga.xml

In the control file ("setup_pymanga.xml") you have to specify the file path of the folder with the OGS input files (line 18), the file with the parameterization of the tree growth model (line 36) and the folder for the output of the model results (line 62). You can also choose an individual file location as output location; it is only important that this folder also exists and is empty - pyMANGA does not automatically create non-existing output locations and does not delete existing files in this.

So an example of the file paths in the order mentioned above might look like the following:

	/home/Dokumente/pyMANGA-master/Benchmarks/Exmouth_Gulf/full_model/TreeOutput

	/home/Dokumente/pyMANGA-master/Benchmarks/Exmouth_Gulf/full_model/Avicennia.py

	/home/Dokumente/pyMANGA-master/Benchmarks/Exmouth_Gulf/full_model

> Groundwater modeling with OGS is performed using an additional Python script. Among other things, this script provides values for the tidal range from the file &bdquo;EXM_Jan-Jul_2019.txt&ldquo; for OGS, puts these data into a loop (since the modeling period is significantly longer than the period of the tidal data) and provides an adjustment of the mean water level. In the script (&bdquo;python_script.py&ldquo;) in line 140 f. the file location of the &bdquo;EXM_Jan-Jul_2019.txt&ldquo;-file must be adapted to the local folder structure.

Reviewer: ```suggestion
Groundwater modeling with OGS is performed using an additional Python script. Among other things, this script provides values for the tidal range from the file &bdquo;EXM_Jan-Jul_2019.txt&ldquo; for OGS, puts these data into a loop (since the modeling period is significantly longer than the period of the tidal data) and provides an adjustment of the mean water level. The file location of the &bdquo;EXM_Jan-Jul_2019.txt&ldquo;-file must be adapted to the local folder structure, in the script (&bdquo;python_script.py&ldquo;) at line 140 f..
```

Author: done
